server/routes/induction/common/qualificationsListController.ts

```diff
@@ -23,7 +23,7 @@ export default abstract class QualificationsListController extends InductionCont
 
     const functionalSkills = {
       ...prisonerFunctionalSkills,
-      assessments: mostRecentAssessments(prisonerFunctionalSkills.assessments),
+      assessments: mostRecentAssessments(prisonerFunctionalSkills.assessments || []),
```

Comment:
This was the fix - when Curious is down the returned `FunctionalSkills` object (`prisonerFunctionalSkills`) has a null `assessments` property, so the code in the `mostRecentAssessments` function errored because it was trying to operate on a `null` object